tests/helper_test.go

```diff
@@ -50,3 +50,64 @@ func TestGetHostNameFailureCase(t *testing.T) {
 	g.Expect(err).To(BeElementOf(errors.New("Fail to get host name: an error")))
```

Comment:
is there a reason this file was added to separate "Tests" folder? Usual go project convention is that /tests/ contains external or integration type tests, and unit tests exist in the folder alongside the code-under-test. Can I move this file into /pkg/utils/ ?

Reply:
moved so we can call private methods